tests/deploy/api/conftest.py

```diff
@@ -0,0 +1,41 @@
+import contextlib
+import json
+
+import httpx
+import pytest
+
+
+@pytest.fixture(scope="package", autouse=True)
+def enhance_raise_for_status(package_mocker):
```

Comment:
I was annoyed by the lack of information while debugging. `response.raise_for_status()` is a nice thing if you just want to check _if_ there is an error. With this, we now automagically also see the content if something fails alongside the status code.